src/ansys/fluent/core/file_session.py

```diff
@@ -126,7 +141,7 @@ def add_scalar_fields_request(
 
         if len(self._file_session._data_file.get_phases()) > 1:
             if not field_name.startswith("phase-"):
-                raise RuntimeError(
+                raise InvalidFieldNamePrefix(
```

Comment:
Again, it's a custom exception type, so why is the caller filling in all the general error information? This makes the approach redundant/pointless.
Also, think about the exception names. Is this really an exception about prefixes or is it actually about field names in multi-phase cases?!

Reply:
Right, it is related to multi-phase prefix error. We will change it to `InvalidMultiPhaseFieldNamePrefix`.

Reply:
Done. Thank you.